apisix/cli/ops.lua

```diff
@@ -728,14 +730,16 @@ local function start(env, ...)
     local pid = util.read_file(pid_path)
     pid = tonumber(pid)
     if pid then
-        local lsof_cmd = "lsof -p " .. pid
-        local res, err = util.execute_cmd(lsof_cmd)
-        if not (res and res == "") then
-            if not res then
-                print(err)
-            else
-                print("APISIX is running...")
-            end
+        local signone = 0
+
+        local ok, err, err_no = signal.kill(pid, signone)
```

Comment:
The CI is failed sometimes: https://github.com/apache/apisix/runs/6399336014?check_suite_focus=true
I guess it is caused by the special meaning of pid -1.

Reply:
You're right. Because of `-1` pid, it always gets success return from [kill()](https://elixir.bootlin.com/linux/latest/source/kernel/signal.c#L3773). Need to change it with another fake pid like `66666`.

Reply:
Changed fake pid, `CLI Test` all the cases passed.

Reply:
I suggest updating the code to pass the existing test instead of changing the test to make refactor pass.

Reply:
Ok, did it return and output invalid tips when negative.